games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -76,19 +72,10 @@ BUILD()
 
  cmake .. \
 	-DCMAKE_BUILD_TYPE=STKRelease \
-	-DCUSTOM_BINDIR="$appsDir/SuperTuxKart" \
 	-DSTK_INSTALL_DATA_DIR=$dataDir/SuperTuxKart \
 	-DSTK_INSTALL_BINARY_DIR=$appsDir \
-	-DCMAKE_INSTALL_PREFIX=$prefix \
-	-DCMAKE_INSTALL_LIBDIR=$libDir \
 	-DBUILD_RECORDER=OFF \
-	-DBUILD_SHARED_LIBS=OFF \
-	-DUSE_CRYPTO_OPENSSL=OFF \
-	-DUSE_IPV6=OFF \
```

Comment:
Keep

Reply:
All default flagging used for Release build.